File: GKPageScrollViewObjc/GKPageScrollViewObjc/Demo/Base/GKDemoBaseViewController.m
//
//  GKDemoBaseViewController.m
//  GKPageScrollView
//
//  Created by QuintGao on 2018/10/27.
//  Copyright © 2018 QuintGao. All rights reserved.
//

#import "GKDemoBaseViewController.h"

@interface GKDemoBaseViewController ()

@end

@implementation GKDemoBaseViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.view.backgroundColor = [UIColor whiteColor];
}

- (BOOL)prefersStatusBarHidden {
    return self.gk_statusBarHidden;
}

- (UIStatusBarStyle)preferredStatusBarStyle {
    return self.gk_statusBarStyle;
}

@end

File: SwiftExample/SwiftExample/Demo/Base/GKDemoBaseViewController.swift
//
//  GKDemoBaseViewController.swift
//  GKPageScrollViewSwift
//
//  Created by QuintGao on 2019/2/21.
//  Copyright © 2019 QuintGao. All rights reserved.
//

import UIKit
import GKNavigationBarSwift
import GKPageScrollView

class GKDemoBaseViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        view.backgroundColor = UIColor.white
    }
    
    deinit {
        print(NSStringFromClass(self.classForCoder) + " deinit")
    }
    
    override var prefersStatusBarHidden: Bool {
        return self.gk_statusBarHidden
    }
    
    override var preferredStatusBarStyle: UIStatusBarStyle {
        return self.gk_statusBarStyle
    }
}
